osmaxx/excerptexport/urls.py

```diff
@@ -3,28 +3,24 @@
 from django.views.generic import TemplateView
 
 from osmaxx.excerptexport.views import (
-    list_downloads,
```

Comment:
Can we delete the code for view `list_downloads` or is it used somewhere besides here (and its own tests)?